distribution/Dockerfile

```diff
@@ -8,9 +8,9 @@ WORKDIR /$BASE_DIR
 
 # ADD FORM distribution
 ADD bin/ /seata-server/bin
-ADD lib/ /seata-server/lib
+ADD target/ /seata-server/target
 ADD conf/ /seata-server/conf
-ADD LICENSE-BIN /seata-server/LICENSE
+ADD LICENSE /seata-server/LICENSE
```

Comment:
Dose there need rename LICENCE-BIN file to LICENCE?

Reply:
Yes, I tried to use `Dockerfile` to build the image, but the log showed that the `LICENSE-BIN` file did not exist.